src/config.rs

```diff
@@ -141,6 +149,7 @@ impl Config {
                 "TOKIO_CONSOLE_LISTEN_ADDRESS",
                 "127.0.0.1:6669".parse().unwrap(),
             )?,
+            enable_upload_metrics: var_optional("ENABLE_UPLOAD_METRICS", true)?,
```

Comment:
The only sort of naming concern that's worth flagging: It wasn't clear to me that this only toggles refreshing the metrics, not their display as well. "FETCH_UPLOAD_METRICS" or "REFRESH_UPLOAD_METRICS" or anything along those lines seem more descriptive.

Reply:
Good point.